doc/fluidsettings.xml

```diff
@@ -452,6 +452,27 @@ Developers:
                 Selects the CoreAudio device to use.
             </desc>
         </setting>
+        <setting>
+            <name>coreaudio.channel-map</name>
+            <type>str</type>
+            <def>0,1</def>
+            <desc>
+                This setting is a comma-separated integer list that maps CoreAudio channels
+                to output device channels. The value of each position in the list, up to the number of
+                output channels available on your device, is the zero-based index of the fluidsynth
+                output channel to route there. Additionally, the special value of -1 will turn off
+                an output.
+
+                For example, the default map for a single stereo output is "0,1". A value of "0,0" will
+                copy the left channel to the right, a value of "1,0" will flip left and right, and a
+                value of "-1,1" will play only the right channel.
+
+                With a six channel output device, and the synth.audio-channels and synth.audio-groups
+                settings both set to "2", a channel map of "-1,-1,0,1,2,3" will result in notes from odd
+                MIDI channels being sent to outputs 3 and 4, and even MIDI channels being send to
+                outputs 5 and 6.
```

Comment:
It took quite some time for me to understand this last part. In case I understood it correctly, I'd suggest to phrase it a bit differently:

```
                This setting is a comma-separated integer list that maps fluidsynth mono-channels
                to CoreAudio device output channels. Each position in the list represents the output channel 
                of the CoreAudio device.
                The value of each position indicates the zero-based index of the fluidsynth
                output mono-channel to route there (i.e. the buffer index used for fluid_synth_process()).
                Additionally, the special value of -1 will turn off an output.

[...your single stereo output example...]

                With a six-channel output device, and the synth.audio-channels and synth.audio-groups
                settings both set to "2", a channel map of "-1,-1,0,1,2,3" will result in notes from odd
                MIDI channels (audible on the first stereo channel, i.e. mono-indices 0,1) being sent to
                outputs 3 and 4, and even MIDI channels (audible on the second stereo channel, i.e. mono-indices 2,3)
                being sent to outputs 5 and 6.

                The list should specify as many positions as there are output channels for the corresponding
                CoreAudio device. Otherwise, the behavior is undefined.
```

Reply:
The updated version has a variation on this. Rather than being undefined, if the list of outputs is too short, it doesn't change the mapping away from the default for the higher channels.

The reasoning for this is that you might have a lot of outputs, and generally don't care about those above however many fluidsynth is producing on due to _synth.audio-channels_.

It makes nice logical sense with the empty string too, in that it's just leaving all channels as they are by default.

Reply:
You haven't applied the first paragraph I suggested. In case that was not intentional, I would amend it after merging, if you don't mind.

Reply:
Thanks, sorry I missed it. Looks great!